test/mrp/README.md

```diff
@@ -0,0 +1,52 @@
+# Kafka-MMITSS Infrastructure Testing Deployment Note
+
+This page indicate a step-by-step instruction to deploy the testing on MMITSS Kafka Transceiver for MMITSS MRP (Road-side processor) components. 
+
+### Requirements
+* cda-mmitss repository cloned
+* confluent_kafka
+* python 3
+* pytest
+* docker compose file(.ymal). Original template can be found here: https://github.com/usdot-fhwa-stol/carma-streets/blob/develop/docker-compose.yml
```

Comment:
Typo yaml

Reply:
Also lets revise the statement here to say `Derived from CARMA Street docker compose <link>`